app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -722,7 +731,11 @@ struct DeviceDelegateOpenXR::State {
   void UpdateClockLevels() {
       if (!OpenXRExtensions::IsExtensionSupported(XR_EXT_PERFORMANCE_SETTINGS_EXTENSION_NAME))
           return;
-
+#ifdef PFDMXR
+      CHECK_XRCMD(OpenXRExtensions::sXrPerfSettingsSetPerformanceLevelEXT(session, XR_PERF_SETTINGS_DOMAIN_CPU_EXT, XR_PERF_SETTINGS_LEVEL_SUSTAINED_HIGH_EXT));
+      CHECK_XRCMD(OpenXRExtensions::sXrPerfSettingsSetPerformanceLevelEXT(session, XR_PERF_SETTINGS_DOMAIN_GPU_EXT, XR_PERF_SETTINGS_LEVEL_SUSTAINED_HIGH_EXT));
+      return;
+#endif
```

Comment:
Hmm, this will likely increase the battery consumption and quickly heat up the device. Is it really needed for the rendering of the browser window and the rest of the UI? Note that for XR and fullscreen videos this will be set anyway.

Reply:
@EasonZxp any comment about what I wrote above? Otherwise I'd have to close this due to lack of activity...